### PR TITLE
Add table aws_health_affected_entity closes #1410

### LINKED ISSUE #1410: Add table aws_health_affected_entity
**References**
https://docs.aws.amazon.com/cli/latest/reference/health/describe-affected-entities.html

### PR DESCRIPTION
**Pull request by ParthaI**
# Integration test logs
<details>
  <summary>Logs</summary>

```
Add passing integration test logs here
```
</details>

# Example query results
<details>
  <summary>Results</summary>

```
Add example SQL query results here (please include the input queries as well)
```
</details>

### CODE REVIEW

**Comment by ParthaI** [comment]
This PR is under review (but difficult to test since we don't have direct access to the API in our accounts).

**Comment by ParthaI** [comment]
Hi @joelcox22 based on your [feedback](https://github.com/turbot/steampipe-plugin-aws/pull/1167#pullrequestreview-1135284503), we have added  `aws_health_affected_entity` in this PR. It would be really appreciated if you could give it a try and share your experience with us. Thanks!

**Comment by joelcox22** [comment]
thanks @ParthaI! I'll be back from a holiday and will be able to take a look at assisting with testing this late next week 😄

File: docs/tables/aws_health_affected_entity.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  event_arn,
+  last_updated_time,
+  status_code
+from
+  aws_health_affected_entity
+where
+  status_code = 'UNIMPAIRED';
+```
+
+### Get health event details for each entity
```
Comment (misraved) [inline docs/tables/aws_health_affected_entity.md:37]:
```suggestion
### Get health event details of each entity
```

**Review by misraved — COMMENTED** [review]
Please take a look at the review comment. Thanks!!

**Review by misraved — APPROVED** [review]

**Comment by misraved** [comment]
@joelcox22 we are pushing this table to the hub for now, please let us know if you have any feedback on this PR in terms of invalid data format or others. You could always raise an issue in the event of failures or suggestions 👍 .

Thanks once again for volunteering to test this table 👍.

**Pull request merged**